src/NexusMods.App.UI/LeftMenu/Downloads/DownloadsLeftMenuDesignViewModel.cs

```diff
@@ -15,7 +15,11 @@ public DownloadsLeftMenuDesignViewModel()
     {
         var items = new ILeftMenuItemViewModel[]
         {
-            new IconViewModel { Name = Language.InProgressTitleTextBlock, Icon = IconType.None },
+            new IconViewModel
+            {
+                Name = Language.InProgressTitleTextBlock,
+                Icon = IconValues.Downloading,
```

Comment:
Should we get these icons from the pages in the future?

Reply:
Yeah probably.